tests/test_deployer.py

```diff
@@ -179,7 +180,8 @@ def test_ingress_is_updated(self, k8s_model, object_types):
                 APPLICATION_NAME,
                 APPLICATION_NAME,
                 SPINNAKER_TAGS,
-                RAW_TAGS
+                RAW_TAGS,
+                ""
```

Comment:
For conformity, could you make the spinnaker_application into a named constant or variable?